### PR TITLE
Added new Warning message KIA0107

### PR DESCRIPTION
**Pull request by hhovsepy**
Issue https://github.com/kiali/kiali/issues/7152
Core PR https://github.com/kiali/kiali/pull/7258

### CODE REVIEW

File: content/en/docs/Features/validations.md [hunk 1]
```diff
@@ -139,6 +139,28 @@ Correct the principal to refer to existing Service Account, make sure that the v
...
 
 
+### KIA0107 - Service Account for this principal found on a remote cluster
+
+AuthorizationPolicy has a Source field, where specifies the source identities of a request.
+In a Source field it accepts the principals, a list of peer identities derived from the peer certificate. The peer identity is in the format of `<TRUST_DOMAIN>/ns/<NAMESPACE>/sa/<SERVICE_ACCOUNT>`, for example, `cluster.local/ns/default/sa/productpage`.
+
+A validation Warning message on a principal value means, that the specified Service Account was found in a cluster different from that of the AuthorizationPolicy.
+
+#### Resolution.
```
Comment (josunect) [inline content/en/docs/Features/validations.md]: Minor thing but, maybe the "." can be removed?

**Review by josunect — COMMENTED** [review]
Just a comment, but I would think on this rule more like an info message, because maybe the user don't want to see a warning if this result that the SPIRE configuration is right. But I guess this will be validated once the SPIRE support is added to Kiali, right?

**Comment by hhovsepy** [comment]
@josunect thanks for the review, I think we can leave it as 'warning' for now as it is temporary before SPIFFY support is implemented.

**Review by josunect — APPROVED** [review]
LGTM

**Pull request merged**